tooltester/src/main/java/io/dockstore/tooltester/runWorkflow/WorkflowRunner.java

```diff
@@ -440,20 +445,45 @@ private void setTaskFamily(final List<String> listOfEcsTasksBeforeNewOneWasAdded
         }
     }
 
-    public void uploadRunInfo() {
-        runMetrics.setExecutionStatus(getExecutionStatus());
+    private void uploadRunInfo() {
+        extendedGa4GhApi.executionMetricsPost(new ExecutionsRequestBody().addRunExecutionsItem(runMetrics), Partner.AGC.name(), getEntryNameForApi(), version, "generated with tooltester ('run-workflows-through-wes' command)");
+    }
 
+    private void saveRunInfo() {
+        String fileName = Instant.now().toEpochMilli() + ".json";
```

Comment:
You could use this helper method from the webservice since it does the same thing https://github.com/dockstore/dockstore/blob/30c353347ee09b06e6aad9e594ba4a07e731675c/dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/S3ClientHelper.java#L148

Reply:
Done